custom-vu/src/main/kotlin/jces1209/vu/ScenarioSimilarities.kt

```diff
@@ -58,6 +53,7 @@ class ScenarioSimilarities(
             editProbability = 0.00f, // 0.10f if we can mutate data
             commentProbability = 0.00f, // 0.04f if we can mutate data
             linkIssueProbability = 0.00f // 0.10f if we can mutate data
+
```

Comment:
I'd avoid unnecessary diff. They generate unnecessary Git conflicts.

Reply:
will take into consideration in the future